CONTRIBUTING.md

```diff
@@ -35,5 +35,6 @@ dotnet build .metadata
 From the root of the repository:
 
 ```sh
-cargo run -p api_gen
+cargo install --git https://github.com/microsoft/windows-rs --rev f1b1edd riddle
+riddle --etc bindings.txt
```

Comment:
For reference, if we were to keep the `api_gen` crate it'd contain the same:

```rust
fn main() -> Result<()> {
    windows_bindgen::bindgen(["--etc", "bindings.txt"])?;
    Ok(())
}
```

And there's no longer the need for `cargo install riddle`.